### PR TITLE
New version: Meltytech.Shotcut version 23.09.29

### PR DESCRIPTION
**Pull request by PckgrBot**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/123412)

### CODE REVIEW

File: manifests/m/Meltytech/Shotcut/23.09.29/Meltytech.Shotcut.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Meltytech.Shotcut
+PackageVersion: 23.09.29
+Platform:
+- Windows.Desktop
+InstallerType: inno
+Scope: machine
```
Comment (Trenly) [inline manifests/m/Meltytech/Shotcut/23.09.29/Meltytech.Shotcut.installer.yaml:9]:
```suggestion
```

File: manifests/m/Meltytech/Shotcut/23.09.29/Meltytech.Shotcut.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/mltframework/shotcut/releases/download/v23.09.29/shotcut-win64-230929.exe
+  InstallerSha256: 197F1F4BC17DA896B5ABBA343F630CD5F69FDBDBD7253BD845C50A17E7D53576
```
Comment (Trenly) [inline manifests/m/Meltytech/Shotcut/23.09.29/Meltytech.Shotcut.installer.yaml:17]:
```suggestion
Installers:
- Architecture: x64
  InstallerUrl: https://github.com/mltframework/shotcut/releases/download/v23.09.29/shotcut-win64-230929.exe
  InstallerSha256: 197F1F4BC17DA896B5ABBA343F630CD5F69FDBDBD7253BD845C50A17E7D53576
  Scope: machine
  InstallerSwitches:
    Custom: /ALLUSERS
- Architecture: x64
  InstallerUrl: https://github.com/mltframework/shotcut/releases/download/v23.09.29/shotcut-win64-230929.exe
  InstallerSha256: 197F1F4BC17DA896B5ABBA343F630CD5F69FDBDBD7253BD845C50A17E7D53576
  Scope: user
  InstallerSwitches:
    Custom: /CURRENTUSER
```

File: manifests/m/Meltytech/Shotcut/23.09.29/Meltytech.Shotcut.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Meltytech.Shotcut
+PackageVersion: 23.09.29
+PackageLocale: en-US
+Publisher: Meltytech, LLC
```
Comment (Trenly) [inline manifests/m/Meltytech/Shotcut/23.09.29/Meltytech.Shotcut.locale.en-US.yaml:7]:
```suggestion
Publisher: Meltytech
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Exception during executable launch operation System.InvalidOperationException: No process is associated with this object. 

This is likely an insignificant error.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**